README.md

```diff
@@ -1,27 +1,77 @@
+
+## Pre-requisites
+
+* Make sure you use at least Node 4
+
+## Installation
+
+```
+npm install --saveDev contentful-sdk-core
+```
+
+## Use case
+
 This package contains some core modules and utilities used by both the [contentful.js](https://github.com/contentful/contentful.js) and [contentful-management.js](https://github.com/contentful/contentful-management.js) SDKs.
 
-## Vendored axios
+## Support
+
+We support all current browsers in their last 3 versions including IE 11. The minimal node version is 4.
 
-The `vendor-browser` and `vendor-node` directories contain a vendored build of [axios](https://github.com/mzabriskie/axios) which are used respectively on the standalone browser build and on the published npm package.
+### ES2015 modules version
 
-Axios is vendored because it expects a native or polyfilled implementation of promises. In this particular case, we vendor axios using babel, which uses the babel-plugin-transform-runtime to transform any usage of promises to requires to `babel-runtime/core-js/promise`.
+For bundlers like webpack2 and rollup we support the `module` &
+`jsnext:main` entry in the package.json
 
-Axios can be vendored with `npm run vendor:browser` and `npm run vendor:node`.
+## Development
 
-The browser vendored version runs on top of the standalone Axios browser version which is already optimized for this use case (it's not a good idea to try and run babel on top of the normal axios commonjs package as it produces an unnecessarily large file)
+### Create the default and the es-modules build:
+```
+npm run build
+```
 
-Because of this, the follow-redirects dependency of axios needs to be a dependency on this package as well, otherwise it won't be installed.
+### Run Tests:
 
-The additional `npm run vendor:version` task is unrelated to axios vendoring and is used to build the library version into the code, to avoid having to bundle code for reading a json file in the browser build.
+Run only the unit tests:
+```
+npm run test:only
+```
 
-## Future incompatibility between lodash and core-js
+Run unit tests including coverage report:
+```
+npm run test:cover
+```
 
-According to this [issue](https://github.com/lodash/lodash/commit/e156459176c1f35964fc443b39f04c4cb0a96763#commitcomment-17483355) in the future lodash and core-js will be incompatible.
+Run unit tests with coverage report and display the result in your browser:
+```
+npm run browser-coverage
+```
 
-This means one of two things:
-- this package will have to stay on a certain version of lodash
-- stop vendoring axios and remove babel-runtime, along with support for node.js 0.10
+Emulate a CI test run:
+```
+npm run test:ci
```

Comment:
isn't `test:ci` for the actual ci, for emulating maybe we can have `test:ci-emulate`